.github/workflows/release.yml

```
@@ -684,6 +684,296 @@ jobs:
+
+          ## 📦 Package Contents
+
+          - \`$binary_pattern\` - Scopes binary for $platform-$arch
+          - \`verification/\` - Security verification files (SHA256 hash + SLSA provenance)
+          - \`sbom/\` - Software Bill of Materials (CycloneDX format)
+          - \`docs/\` - Installation and security documentation
+
+          ## 🚀 Quick Installation
+
+          ### For $platform
+EOF
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 773: could not find expected ':'

(syntax-check)